certbot-apache/certbot_apache/parser.py

```diff
@@ -840,7 +841,12 @@ def parsed_in_current(self, filep):
         :returns: True if file is parsed in existing configuration tree
         :rtype: bool
         """
-        return self._parsed_by_parser_paths(filep, self.parser_paths)
+        paths = []
+        for directory in self.parser_paths:
```

Comment:
since this code is duplicated, it seems best to keep it factored out as before, no?

Reply:
Yeah, sounds good. I created a helper function to reuse in both of the cases.